creativecloud/scripts/utils.js

```diff
@@ -61,7 +61,7 @@ function getDecorateAreaFn() {
   function replaceDotMedia(area = document) {
     const currUrl = new URL(window.location);
     const pathSeg = currUrl.pathname.split('/').length;
-    if (pathSeg >= 3) return;
+    if (pathSeg >= 4) return;
```

Comment:
Thanks @seanchoi-dev for fixing this. Functionality wise this looks good. However this update will also update media paths for all our product pages like https://www.adobe.com/products/illustrator.html Initially we were going to load all our media paths from contentRoot(cc-shared). However then team noticed some performance impact and that's why restricted it to only top level pages. 

Can we put a check to see 4th path segment is in format of locale (** or **_**) to update the media path. So that product pages performance is not impacted.

More discussion can be seen in this ticket https://jira.corp.adobe.com/browse/MWPW-142577 about performance concern that team had identified.

cc @aishwaryamathuria @salonijain3